models/sales/sales.yml

```
@@ -59,7 +59,60 @@ models:
       - name: owner_full_name
         description: The full name of the deal's owner.
 
-        
+  - name: hubspot__deal_stages
+    description: Each record represents a stage within a deal in Hubspot
+    columns:
+      - name: deal_stage_id
+        description: The unique deal stage identifier.
+        tests:
+          - not_null
+          - unique
+
+      - name: deal_stage_name
+        description: The name of the deal stage.
+
+      - name: date_stage_entered
+        description: The timestamp the deal stage was entered.
+
+      - name: date_stage_exited
+        description: The Fivetran calculated exist time of the deal stage.
```

is this supposed to be 
```suggestion
        description: The Fivetran calculated exit time of the deal stage.
```